code/game/gamemodes/gang/gang_items.dm

```diff
@@ -267,20 +460,74 @@
 	cost = 40
 	item_path = /obj/item/ammo_box/magazine/uzim9mm
 
+/datum/gang_item/weapon/launcher
+	name = "88mm high explosive rocket launcher"
```

Comment:
Its a sleeper UP item. Basically a ranged welderbomb (wider radius but only light explosion) that can only detonate on dense objects. Also 3 seconds to aim. We'll see though.